tests/suite/cli_paths.rs

```diff
@@ -362,7 +362,8 @@ mod windows {
     fn install_uninstall_affect_path() {
         clitools::test(Scenario::Empty, &|config| {
             with_saved_path(&mut || {
-                let path = format!("{:?}", config.cargodir.join("bin").to_string_lossy());
+                let path = format!("{:?}", config.cargodir.join("bin").to_string_lossy())
```

Comment:
Can you give a bit or context? How did the winreg bump change the `config.cargodir` value and why should we fix it in this test instead of at some other level?

Reply:
@djc This test depends on https://togithub.com/gentoo90/winreg-rs/issues/52 which is a breaking change for formatting.

Reply:
So should we switch to formatting with `{}` instead of `{:?}` like upstream, instead? Even after looking at the upstream issue, this fix looks more like a workaround to me.

Reply:
@djc Indeed, I have no idea why the previous code was printing a string with `Debug`.

Reply:
@djc I changed that to use `config.cargodir.join("bin").display()` instead. The point here is that a path cannot be `Display`ed directly, and if I use `Debug` it'll have double backslashes.